test/Microsoft.ApplicationInsights.AspNetCore.Tests/project.json

```diff
@@ -9,36 +9,37 @@
     }
   },
   "dependencies": {
-    "Microsoft.AspNetCore.Http.Features": "1.0.0-rc2-final",
-    "Microsoft.AspNetCore.Http": "1.0.0-rc2-final",
-    "Microsoft.AspNetCore.Http.Extensions": "1.0.0-rc2-final",
-    "Microsoft.AspNetCore.Hosting": "1.0.0-rc2-final",
-    "Microsoft.AspNetCore.Mvc.Core": "1.0.0-rc2-final",
-    "Microsoft.Extensions.Logging": "1.0.0-rc2-final",
-    "Microsoft.Extensions.Configuration.Json": "1.0.0-rc2-final",
     "xunit": "2.1.0",
-    "dotnet-test-xunit": "1.0.0-*",
     "Microsoft.ApplicationInsights.AspNetCore": { "target": "project" },
-    "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0-rc2-final"
+    "dotnet-test-xunit": "1.0.0-rc3-*",
+    "Microsoft.AspNetCore.Hosting": "1.0.0",
+    "Microsoft.AspNetCore.Http": "1.0.0",
+    "Microsoft.AspNetCore.Http.Extensions": "1.0.0",
+    "Microsoft.AspNetCore.Http.Features": "1.0.0",
+    "Microsoft.AspNetCore.Mvc.Core": "1.0.0",
+    "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0",
+    "Microsoft.Extensions.Configuration.Json": "1.0.0",
+    "Microsoft.Extensions.Logging": "1.0.0"
   },
   "testRunner": "xunit",
   "frameworks": {
     "netcoreapp1.0": {
       "imports": [
         "dnxcore50",
         "dotnet5.4",
-        "portable-net451+win8"
+        "portable-net451+win8",
+        "portable-dnxcore50+net45+win8+wp8+wpa81"
```

Comment:
Shouldn't be needed, I think it is left over from trying to get the RC3 stuff working.